lte/gateway/python/magma/mobilityd/ip_allocator_pool.py

```diff
@@ -230,7 +231,10 @@ def alloc_ip_address(self, sid: str, vlan_id: int) -> IPDesc:
             ip_desc.state = IPState.ALLOCATED
             return ip_desc
         else:
-            logging.error("Run out of available IP addresses")
+            logging.error(
+                "Run out of available IP addresses",
+                extra=EXCLUDE_FROM_ERROR_MONITORING,
```

Comment:
This is legitimate error that should be reported by sentry so that operator can fix the configuration. why is it excluded?

Reply:
+1 I think this should not be excluded

Reply:
Normally we try to exclude application errors that don't indicate something wrong in the code and are more related to server maintenance. In this case, it's not a very frequent error so we can also leave it if you prefer.